### PR TITLE
Sky + MK Bio

### PR DESCRIPTION
**Pull request by atechnicolorskye**

### CODE REVIEW

File: content/authors/si-kai-lee/_index.md [hunk 1]
```diff
@@ -41,7 +42,7 @@ education:
 social:
 - icon: envelope
   icon_pack: fas
-  link: '#contact'  # For a direct email link, use "mailto:test@example.org".
+  link: 'mailto:sikai.lee@chicagobooth.edu'
 - icon: twitter
   icon_pack: fab
   link: https://twitter.com/USERNAME
```
Comment (mlakolar) [inline content/authors/si-kai-lee/_index.md:49]: You can delete this if you do not have account

File: netlify.toml [hunk 2]
```diff
@@ -3,7 +3,7 @@
   publish = "public"
 
 [build.environment]
-  HUGO_VERSION = "0.68.3"
+  HUGO_VERSION = "0.62.0"
```
Comment (mlakolar) [inline netlify.toml]: Should we keep this at 68.3 ?

File: content/authors/si-kai-lee/_index.md [hunk 3]
```diff
@@ -50,7 +51,7 @@ social:
   link: https://scholar.google.com/citations?user=PERSON-ID
```
Comment (mlakolar) [inline content/authors/si-kai-lee/_index.md:52]: Do you have google scholar account? You can add dblp link and linked in as well I think...

Comment (atechnicolorskye) [reply content/authors/si-kai-lee/_index.md]: I don't have that, but added LinkedIn, you should pub this

**Pull request merged**